compiler/dyno/include/chpl/uast/Attributes.h

```diff
@@ -40,19 +40,28 @@ class Attributes final : public AstNode {
  private:
   std::set<PragmaTag> pragmas_;
   bool isDeprecated_;
+  bool isUnstable_; //new
   UniqueString deprecationMessage_;
+  UniqueString unstableMessage_;
 
   // TODO: Do we want to preserve the location of string literals used in
   // pragmas and deprecation messages?
-  Attributes(std::set<PragmaTag> pragmas, bool isDeprecated,
-             UniqueString deprecationMessage)
+  Attributes(std::set<PragmaTag> pragmas, bool isDeprecated,  //add new parameters for unstable -- must update new attributes in other folders as well to take in same # of parameters
+             UniqueString deprecationMessage,
```

Comment:
I think this is the line that's causing that error I mentioned - the ordering of arguments here should match the ordering of fields above, so it should be:
```
bool isDeprecated_,
bool isUnstable_,
UniqueString deprecationMessage,
UniqueString unstableMessage)
```